### PR TITLE
Housekeeping/Create and fail destination plates

### PR DESCRIPTION
**Pull request by pjvv**
Changes proposed in this pull request:

* update API key
* fix response schema

### CODE REVIEW

File: modules/lighthouse_service_biosero.js [hunk 1]
```diff
@@ -20,34 +20,76 @@ const createDestinationPlateBiosero = async (form) => {
...
+      // successfully created the event and performed subsequent processes
       return {
         success: true,
         response: `Successfully created destination plate with barcode: ${form.barcode}`,
       }
     } else {
       // contains status code and message
-      const errors = response._error
-      // failure
+      const error = response.data._error
```
Comment (andrewsparkes) [inline modules/lighthouse_service_biosero.js:79]: Given the example responses above, I don't get where the 'data' level comes from here, or in the catch further down.

File: pages/biosero_cherrypick.vue [hunk 2]
```diff
@@ -86,15 +86,10 @@ export default {
...
       let message, type
       if (response.success) {
-        if (response.errors) {
-          message = response.errors.join(', ')
-          type = 'warning'
-        } else {
-          message = response.response
-          type = 'success'
-        }
+        message = response.response
```
Comment (andrewsparkes) [inline pages/biosero_cherrypick.vue:89]: response.response? is that right?

**Review by andrewsparkes — APPROVED** [review]

File: modules/lighthouse_service_biosero.js (hunk 1 above)
Comment (pjvv) [reply modules/lighthouse_service_biosero.js:79]: From axios: https://github.com/axios/axios#response-schema

File: pages/biosero_cherrypick.vue (hunk 2 above)
Comment (pjvv) [reply pages/biosero_cherrypick.vue:89]: Yeah it's ugly but kept the same name as I couldn't think of a better name during the refactor. :P

**Pull request merged**